packages/components/src/index.ts

```diff
@@ -17,7 +17,11 @@ export {
 	default as Animate,
 	getAnimateClassName as __unstableGetAnimateClassName,
 } from './animate';
-export { __unstableMotion, __unstableAnimatePresence } from './animation';
+export {
+	__unstableMotion,
+	__unstableAnimatePresence,
+	__unstableMotionContext,
```

Comment:
TODO: we may want to export the motion context differently? (ie. stable, or as private API)

Reply:
Let's export all the providers (style, components, toolbar context, motion) consistently the same way, as private APIs.

Reply:
I'm not sure we can remove existing exports from the package — even if they are prefixed with `__unstable`, removing them would cause breaking changes in plugin and themes using them (I ran [a quick search on wpdirectory](https://wpdirectory.net/search/01H2GD26S6CXSS8CWX4PWN771G) and there are unfortunately some matches).

@youknowriad do you have any suggestions on how we may want to approach this? Given how "de facto" stable are `__unstableMotion` and `__unstableAnimatePresence`, should we just export them without the `__unstable` alias too?